app/models/metadata_presenter/grid.rb

```diff
@@ -225,6 +231,12 @@ def detached?(position)
       position[:row].nil? || position[:column].nil?
     end
 
+    def confirmation_and_cya_pages_present?
+      service.confirmation_page.present? &&
+        service.checkanswers_page.present? &&
+        !coordinates.positions[service.checkanswers_page.uuid][:column].nil?
```

Comment:
```suggestion
        coordinates.positions[service.checkanswers_page.uuid][:column].present?
```

Reply:
if only the confirmation page is present i assume the grid just places the confirmation page wherever the column model calculated it?

Reply:
yep correct, the grid will place the confirmation page where the column model has calculated.